### DISCUSSION TITLE
Installation Problem

### DISCUSSION THREAD

**Discussion by benhartwich** [AllskyTeam/allsky-modules#156] [Q&A] [answered]
Hi,
I would like to install the modules with the user allsky, but this user can´t solve the variable for allsky home correctly, which is /home/allsky/allsky/ instead of /home/allsky. I get the error: 
`sudo -u allsky ./install.sh
./install.sh: line 6: /home/allsky/variables.sh: No such file or directory
./install.sh: line 6: exit: : numeric argument required
`
How could I solve it. With root everything works fine, but the installation doesn´t continue because of root.

**Comment by Alex-developer** [comment, marked as answer]
Its best to install the modules using the same user you installed allsky with.

Alternatively do the following before running the installer, ensuring you are the right user  i.e. allsky

export ALLSKY_HOME=/home/allsky/allsky

**Reply by benhartwich** [reply]
Thanks, I've edited the install.sh and set the variable to the right path.
